src/resources/ModifierTemplates/ModifierTemplateInterfaces.ts

```diff
@@ -0,0 +1,76 @@
+export interface ModifierModel {
+    /**
+     * Unique identifier
+     */
+    id: string;
+    /**
+     * English human friendly name
+     */
+    displayName: string;
+    /**
+     * Category the modifier belongs to.
+     * LICENSE and SYSTEM are more for internal use and might need to be filtered out from display
+     */
+    modifierType: ModifierType;
+    /**
+     * List of statements contained in the modifier
+     */
+    statements: StatementModel[];
+}
+
+export interface StatementModel {
```

Comment:
We already have a StatementModel from another part of the app. It will break if you try to import it in the admin-ui

https://github.com/coveo/platform-client/blob/a1ea925ffcce1aeaf5c419b2dea6e753b76548fc/src/resources/Pipelines/Statements/StatementsInterfaces.ts

I suggest renaming to ModifierStatementModel